src/routes/controllers/articles.controller.js

```diff
@@ -261,3 +262,41 @@ export async function shareArticleViaEmail(req, res) {
     );
   }
 }
+
+/**
+ *
+ *
+ * @export
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} report an article
+ */
+export async function reportArticle(req, res) {
+  const { articleId } = req.params;
+  const { user } = req;
+  const { reportType, content } = req.body;
+
+  const article = await getArticlebyId(articleId);
+  if (!article) return errorResponse(res, 404, 'Article does not exist', true);
+
+  const reportedArticle = await Report.create({
```

Comment:
What happens if this throws an error?

Reply:
I have implemented this feedback and catered for it in a catch block.